.circleci/unittest/linux/scripts/setup_env.sh

```diff
@@ -86,3 +86,7 @@ fi
 pip install pip --upgrade
 
 conda env update --file "${this_dir}/environment.yml" --prune
+
+if [[ $os == 'Linux' ]]; then
+  pip install envpool
```

Comment:
For lib such as this one we generally do a single test in a dedicated pipeline of the file test_libs
Have a look at the jumanji pipeline for instance